lib/rubocop/cop/variable_force/assignment.rb

```diff
@@ -109,8 +109,13 @@ def operator_assignment_node
           node.parent
         end
 
+        # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
         def multiple_assignment_node
-          return nil unless node.parent&.mlhs_type?
+          candidate_mlhs_node = node&.parent
+          # In `(foo, bar), *baz`, the splat node must be traversed as well.
+          candidate_mlhs_node = candidate_mlhs_node&.parent if candidate_mlhs_node&.splat_type?
+
+          return nil unless candidate_mlhs_node&.mlhs_type?
           return nil unless (grandparent_node = node.parent&.parent)
```

Comment:
By suppressing safe navigation, the new `rubocop:disable` annotation will likely no longer be necessary.

```suggestion
          return unless (candidate_mlhs_node = node.parent)

          # In `(foo, bar), *baz`, the splat node must be traversed as well.
          candidate_mlhs_node = candidate_mlhs_node.parent if candidate_mlhs_node.splat_type?

          return nil unless candidate_mlhs_node.mlhs_type?
          return nil unless (grandparent_node = node.parent.parent)
```

Reply:
Nice, thanks. It makes it a bit better, but one is still there.